src/app/devices/deviceContent/components/moduleIdentity/moduleIdentity.tsx

```diff
@@ -9,11 +9,12 @@ import { CommandBar } from 'office-ui-fabric-react/lib/CommandBar';
 import { LocalizationContextConsumer, LocalizationContextInterface } from '../../../../shared/contexts/localizationContext';
 import { ResourceKeys } from '../../../../../localization/resourceKeys';
 import { getDeviceIdFromQueryString } from '../../../../shared/utils/queryStringHelper';
-import { REFRESH } from '../../../../constants/iconNames';
+import { REFRESH, ArrayOperation } from '../../../../constants/iconNames';
```

Comment:
Constants should use SCREAMING_SNAKE_CASE

Reply:
Disregard. I just noticed this is an enum and not a single, static value.

Reply:
:) Sure. I've just merged master in.